app/views/devise/two_factor_authentication/show.html.slim

```diff
@@ -8,6 +8,8 @@ p.mt-tiny.mb0#2fa-description
     url: otp_send_path,
     method: :get,
     role: 'form', class: 'mt3') do |f|
+  - if reauthn?
+    = f.input :reauthn, as: :hidden, input_html: { value: 'true' }
```

Comment:
alternatively, we could always have this hidden field and have the value be `reauthn?` Then we wouldn't have to check for `reauthn.present?` in the controller, just for its value.